editor.planx.uk/src/components/EditorNavMenu.tsx

```diff
@@ -243,11 +242,7 @@ function EditorNavMenu() {
 
   const { routes, compact } = getRoutesForUrl(url.href);
 
-  const visibleRoutes = routes.filter(({ accessibleBy }) => {
-    if (user?.isPlatformAdmin) return accessibleBy.includes("platformAdmin");
-    if (canUserEditTeam(teamSlug)) return accessibleBy.includes("teamEditor");
-    return accessibleBy.includes("teamViewer");
-  });
+  const visibleRoutes = routes.filter(({ accessibleBy }) => role && accessibleBy.includes(role));
```

Comment:
This is checking they can only see routes with "demoUser" in the `accessibleBy` rather than restricting the paths they shouldn't see?

Reply:
Yes that's right - this is just a visual control for the menu 👍